main.js

```diff
@@ -1,11 +1,18 @@
 import adjectives from './data/adjectives.js'
 import nouns from './data/nouns.js'
 
-export function generate_name(maxNum = 0) {
+export function generate_name(maxNum = 0, separator = '') {
     const adjective = adjectives[Math.floor(Math.random() * adjectives.length)]
     const noun = nouns[Math.floor(Math.random() * nouns.length)]
+    const randomNumber = Math.floor(Math.random() * maxNum + 1)
+
+    if (separator) {
```

Comment:
Thanks for the pull request! This does look a redundant with two ternary operator statements. Maybe pull out the strings to an array and then operate on it?

Reply:
Thank you for reviewing my code! I reduced the code to just a return statement by using arrays as you suggested.